kubernetes-docs/spbm.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Storage Policy Based Management for dynamic provisioning of volumes
+---
+
+## Overview
+
+One of the most important features of vSphere for Storage Management is Policy based Management. Storage Policy Based Management (SPBM) is a storage policy framework that provides a single unified control plane across a broad range of data services and storage solutions. SPBM enables vSphere administrators to overcome upfront storage provisioning challenges, such as capacity planning, differentiated service levels and managing capacity headroom
+ 
+As we discussed in previously StorageClass specifies provisioner and parameters. And using these parameters you can define the policy for that particular PV which will be dynamically provisioned. 
+ 
+You can specify the existing vCenter Storage Policy Based Management (SPBM) policy to configure a persistent volume with SPBM policy. storagePolicyName parameter is used for this.
+ 
+**Note: SPBM policy based provisioning of persistent volumes will be available in 1.7.x release.**
+
+**Note:**
+
+All the example yamls can be found [here](https://github.com/kubernetes/kubernetes/tree/master/examples/volumes/vsphere) unless otherwise specified. Please download these examples.
+
+**Create Storage Class**
```

Comment:
You have missed one more YAML from SPBM section in https://github.com/kubernetes/kubernetes/tree/master/examples/volumes/vsphere#storage-class.